audio-analysis/lib/etl/sqlConnection.py

```diff
@@ -60,17 +77,19 @@ def sqlConnectionSetup(engine):
                     Column('pitch',Float),
                     Column('p_confidence',Float),
                 )
-        # Create all tables
-        meta.create_all()
-
-#Function checks if url exists in table already
-def urlExists(url, engine):
-    # Check if URL already exists
-    result = engine.execute('SELECT * FROM `test_table` WHERE `url`="' + url + '"')
-    print("Number of rows in database for this video", result.rowcount)
+    elif table_name == 'mfcc_test_table':
+        mfcc_table = Table(table_name,meta,
+                    Column('mfcc_id',Integer, primary_key=True, autoincrement=True),
+                    Column('index',Integer),
+                    Column('end_time_s',Integer),
+                    Column('start_time_s',Integer),
+                    Column('url',String(length=500)),
+                )
+        for i in range (1,60):
+            engine.execute('ALTER TABLE `'+ table_name +'` ADD mfcc_'+ '{0:0=2d}'.format(i) +' float;')
```

Comment:
What does this do?

Reply:
This loops through the MFCC table and adds the appropriate columns (mfcc_1 to mfcc_40).